### DISCUSSION TITLE
remove restricted people from beatmap leaderboard

### DISCUSSION THREAD

**Discussion by AzusaNakano2** [ppy/osu#22473] [Ideas] [answered]
i've found beatmap that banned or restricted user are still on leaderboard, it should be good to remove them?

![image](https://user-images.githubusercontent.com/114277426/215943687-7374ff21-937b-4c00-b99e-5bc15f080b25.png)
https://osu.ppy.sh/beatmapsets/1054646#fruits/2203620
https://osu.ppy.sh/scores/fruits/188856844

and 

![screenshot1919](https://user-images.githubusercontent.com/114277426/215943919-c52ba87d-0bfb-4a5b-a828-466a16df18e8.jpg)
https://osu.ppy.sh/beatmapsets/1318691#fruits/2731828
https://osu.ppy.sh/scores/fruits/193339399

**Comment by cdwcgt** [comment, marked as answer]
These users should have submitted account deletion requests. I remember that account deletion will not delete(or hide) these data.
for restricted user, there score will be hide.
https://osu.ppy.sh/legal/en/Privacy#your-rights-and-control

`
As we have a strict one-account-per-user policy to maintain fair leaderboards, after account deletion you may not be able to return to our service. To enforce this, we will also maintain the unique identifying string as mentioned in "Information we collect", even after account deletion. This is done so as a legitimate interest as outlined in the GDPR in order to enforce our rules.
`
